src/http.js

```diff
@@ -207,11 +204,48 @@ function formatValue(input, skipEncoding) {
     return value.map(encodeFn).join(',')
   }
   if (collectionFormat === 'multi') {
+    // query case (not multipart/formdata)
     return value.map(encodeFn)
   }
   return value.map(encodeFn).join(SEPARATORS[collectionFormat])
 }
 
+function buildFormData(reqForm) {
```

Comment:
Seems that algorithm is duplicated on two places and cyclomatic complexity is unnecessarily high. Let's try to do something different about it e.g. using reducer

```js
function buildFormData(reqForm) {
  return Object.entries(reqForm).reduce((formData, [name, input]) => {
     if (!input.collectionFormat && !input.isOAS3formatArray) {
       formData.append(name, formatValue(input, true));
     } else {
       input.value.forEach(item => formData.append(name, formatValue({...input, value: item}, true)));
     }     
     return formData;
  }, new FormData());
}

Reply:
refactor implemented.